torch/_torch_docs.py

```diff
@@ -6819,10 +6875,12 @@ def merge_dicts(*dicts):
            r"""
 sparse_coo_tensor(indices, values, size=None, *, dtype=None, device=None, requires_grad=False) -> Tensor
 
-Constructs a sparse tensors in COO(rdinate) format with non-zero elements at the given :attr:`indices`
-with the given :attr:`values`. A sparse tensor can be `uncoalesced`, in that case, there are duplicate
-coordinates in the indices, and the value at that index is the sum of all duplicate value entries:
-`torch.sparse`_.
+Constructs a :ref:`sparse tensors in COO(rdinate) format
+<sparse-coo-docs>` with specified elements at the given
```

Comment:
"with the specified values at the given indices."

Reply:
fixed.